### PR TITLE
The ``salt-run`` CLI now accepts ``--jid`` which allows scripting against it

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title.

This helps with proper testing under PyTest

### CODE REVIEW

**Review by krionbsd — DISMISSED** [review]

File: doc/topics/releases/3003.rst [hunk 1]
```diff
@@ -23,3 +23,4 @@ Added
 -----
 
 - Firewall groups support to Vultr Salt Cloud provider
+- The ``salt-run`` CLI now accepts ``--jid`` which allows scripting against it.
```
Comment (Ch3LL) [inline doc/topics/releases/3003.rst:26]: Changelog file is needed as well

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: doc/topics/releases/3003.rst (hunk 1 above)
Comment (s0undt3ch) [reply doc/topics/releases/3003.rst:26]: K. I'll create an issue for this since we need to link to one

Comment (s0undt3ch) [reply doc/topics/releases/3003.rst:26]: Done

**Review by Ch3LL — APPROVED** [review]

**Comment by s0undt3ch** [comment]
These windows test failure are now a know issue and unrelated to the changes to this PR as seen in https://github.com/saltstack/salt/pull/59530.
It's being tracked in https://github.com/saltstack/salt/issues/59531

**Review by krionbsd — APPROVED** [review]

**Review by twangboy — APPROVED** [review]
Still failing tests on Windows

**Comment by s0undt3ch** [comment]
@twangboy, yes, see https://github.com/saltstack/salt/pull/59530 and https://github.com/saltstack/salt/issues/59531

**Pull request merged**